src/api/forms/repositories/form-metadata-repository.js

```diff
@@ -180,6 +208,6 @@ export async function remove(formId, session) {
 
 /**
  * @import { FormMetadataDocument } from '@defra/forms-model'
- * @import { ClientSession, Collection, UpdateFilter } from 'mongodb'
- * @import { PartialFormMetadataDocument } from '~/src/api/types.js'
+ * @import { ClientSession, Collection, UpdateFilter, WithId } from 'mongodb'
+ * @import { PartialFormMetadataDocument, RequiredPaginationOptions } from '~/src/api/types.js'
```

Comment:
I can't see `RequiredPaginationOptions`?

```suggestion
 * @import { PartialFormMetadataDocument, PaginationOptions } from '~/src/api/types.js'
```

Reply:
Good spot, that doesn't exist anymore.